### ISSUE TITLE
All issues seem to be missing

### ISSUE THREAD

**Issue by Gaibhne** [getferdi/ferdi#2479]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/getferdi/ferdi/issues) for a bug report similar to mine, in vain

### Steps to reproduce

I distinctly remember reporting issues in the last few days, and upon checking on their status, they all seem to be gone :(

### Debug link

Doesn't apply

**Comment by Gaboris** [comment]
Yeah, seems like the backtrack on the "If you don't pay you don't actually deserve to have issues" idiocy only applies as long as the team reads the bug report and not meant for "community engagement" aka "this app isn't flawless and you need workarounds".

**Comment by Gaibhne** [comment]
I'm not really sure what that means. It sounds like it's referencing some drama, but without context I'm not even sure if it's pro or con 🤷

**Comment by Gaboris** [comment]
Right, sorry. It's sadly drama that went down a few months ago.

The dev team seemingly went through some fundamental shifts that caused multiple old and important members to leave. After which the "new" team disabled all Issues on this repo claiming "It's for the users' security as they kept sharing personal info." and made it only accessible to donating members.

A few weeks later they stepped back on this decision and opened up Issues again, but as you can see they systematically purge them. Either so no one can see how many problems the app still has or to try and push people to donate to SEE potential fixes mentioned by other users.

It's a very unfortunate situation when this app has introduced so many improvements Franz's base code which has actually just recently started up development again with some big updates. 👍

**Comment by spacecabbie** [comment]
> Right, sorry. It's sadly drama that went down a few months ago.
> 
> The dev team seemingly went through some fundamental shifts that caused multiple old and important members to leave. After which the "new" team disabled all Issues on this repo claiming "It's for the users' security as they kept sharing personal info." and made it only accessible to donating members.
> 
> A few weeks later they stepped back on this decision and opened up Issues again, but as you can see they systematically purge them. Either so no one can see how many problems the app still has or to try and push people to donate to SEE potential fixes mentioned by other users.
> 
> It's a very unfortunate situation when this app has introduced so many improvements Franz's base code which has actually just recently started up development again with some big updates. 👍

It seems that the portable version at least is instabile I would not be surprised that minorannoyances have been introduced to promote subscription to issue tracker.
